readme.md

```diff
@@ -101,6 +101,10 @@ While we recommend you use our widgets for your storefront, we do have a shortco
 ### How do I make the domain search return results on page load? ###
 You can add `?domainToCheck=example.com` to your query string on any page that has the domain search widget and the widget will perform an automatic search on page load.
 
+### Getting Started
+
+To get a working WordPress environment please follow our [Getting Started: Docker Runbook guide.](getting-started.md)
```

Comment:
the readme.md should not be edited directly.  only the readme.txt
Also I don't advise putting this here because it will confuse the user of the plugin because this will show up in the wordpress.org repository.